source/tutorials.rst

```diff
@@ -157,6 +248,11 @@ The OSWEC was selected because its design is fundamentally different from the RM
 | -3.9 |         |            |
 +------+---------+------------+
 
+.. note::
+	Again, I assume the rows refer to x, y, z. It might also be useful to note
+	where the origin is. Is it not better to have the origin at the base of the
+	device, given its motion?
+
```

Comment:
We provide the coordinate system information here: http://wec-sim.github.io/WEC-Sim/theory.html#coordinate-system but if you have a suggestion on how to improve the tutorial descriptions, please submit a PR

Reply:
See #456